storm-core/test/clj/backtype/storm/security/auth/nimbus_auth_test.clj

```diff
@@ -48,6 +48,7 @@
         nimbus-server (ThriftServer. (:daemon-conf cluster-map)
                                      (Nimbus$Processor. (:nimbus cluster-map)) 
                                      ThriftConnectionType/NIMBUS)]
+    (Thread/sleep 2000)
```

Comment:
Sleeps in tests are problematic.  Is there a better way that we can check if we are ready instead of just waiting?